### PR TITLE
Make the links work

### PR DESCRIPTION
**Pull request by matthiasclasen**
It turns out that the syntax for links is [title](url), not [url](title).
Fix this mixup throughout the document.

### CODE REVIEW

**Comment by matthiasclasen** [comment]
Should have known that github would interpret the markdown in my commit message. Oh well.

File: source/docs/fedora-atomic-workstation.md [hunk 1]
```diff
@@ -18,14 +18,14 @@ dd if=yourisoname.iso of=/dev/sdb bs=4M
 
 **Installing inside an existing system**
 
-If you rather want to install FAW inside an existing system, please follow the steps described on [https://pagure.io/workstation-ostree-config/blob/master/f/README-install-inside.md](Pagure).
+If you rather want to install FAW inside an existing system, please follow the steps described on [Pagure](https://pagure.io/workstation-ostree-config/blob/master/f/README-install-inside.md).
 
 ## Support
 
-For now, you can ask us questions in the #atomic channel on IRC Freenode or via Twitter [https://twitter.com/projectatomic](@projectatomic). You can also use the [](atomic-devel) mailing list. Contributions are welcome.
+For now, you can ask us questions in the #atomic channel on IRC Freenode or via Twitter [@projectatomic](https://twitter.com/projectatomic). You can also use the [atomic-devel]() mailing list. Contributions are welcome.
```
Comment (jlebon) [inline source/docs/fedora-atomic-workstation.md]: Not new, though it looks like this is missing a link. Probably meant to link to https://lists.projectatomic.io/mailman/listinfo/atomic-devel? Feel free to ignore, we can do that as a follow up as well. :)

File: source/docs/fedora-atomic-workstation.md [hunk 2]
```diff
@@ -18,14 +18,14 @@ dd if=yourisoname.iso of=/dev/sdb bs=4M
...
 
-We are also in the process of creating a [https://fedoraproject.org/wiki/Workstation/AtomicWorkstation/SIG](Special Interest Group) - if you are interested in joining, let us know via one of the mentioned channels and we can add you.
+We are also in the process of creating a [Special Interest Group](https://fedoraproject.org/wiki/Workstation/AtomicWorkstation/SIG) - if you are interested in joining, let us know via one of the mentioned channels and we can add you.
 
 ## Updating and Reverting Fedora Atomic Workstation
 
-For information on how to update and rollback your system, please read [http://www.projectatomic.io/docs/os-updates/](Operating System Upgrades via rpm-ostree).
+For information on how to update and rollback your system, please read [Operation System Upgrades via rpm-ostree](http://www.projectatomic.io/docs/os-updates/).
```
Comment (dustymabe) [inline source/docs/fedora-atomic-workstation.md]: can you change this to `Operating` again?

File: source/docs/fedora-atomic-workstation.md (hunk 1 above)
Comment (dustymabe) [reply source/docs/fedora-atomic-workstation.md]: that is taken care of in #520

File: source/docs/fedora-atomic-workstation.md [hunk 3]
```diff
@@ -4,11 +4,11 @@ HowTo: Fedora Atomic Workstation
 
 More detailed documentation with cheat sheet is coming on 15 February 2018.
 
-For a summary of the why and how, please read [https://fedoraproject.org/wiki/Workstation/AtomicWorkstation](Atomic Workstation) and [https://fedoraproject.org/wiki/Changes/WorkstationOstree](Workstation OSTree).
+For a summary of the why and how, please read [Atomic Workstation](https://fedoraproject.org/wiki/Workstation/AtomicWorkstation) and [Workstation OSTree](https://fedoraproject.org/wiki/Changes/WorkstationOstree).
 
 ## Quickstart
 
-Download the [https://dl.fedoraproject.org/pub/fedora/linux/releases/27/WorkstationOstree/x86_64/iso/](.iso) and use it to create installation media.
+Download the [.iso](https://dl.fedoraproject.org/pub/fedora/linux/releases/27/WorkstationOstree/x86_64/iso/) and use it to create installation media.
```
Comment (dustymabe) [inline source/docs/fedora-atomic-workstation.md]: can you change the `dl.fedoraproject` to `download.fedoraproject` ?

File: source/docs/fedora-atomic-workstation.md [hunk 4]
```diff
@@ -18,14 +18,14 @@ dd if=yourisoname.iso of=/dev/sdb bs=4M
 
 **Installing inside an existing system**
 
-If you rather want to install FAW inside an existing system, please follow the steps described on [https://pagure.io/workstation-ostree-config/blob/master/f/README-install-inside.md](Pagure).
+If you rather want to install FAW inside an existing system, please follow the steps described on [Pagure](https://pagure.io/workstation-ostree-config/blob/master/f/README-install-inside.md).
 
 ## Support
 
-For now, you can ask us questions in the #atomic channel on IRC Freenode or via Twitter [https://twitter.com/projectatomic](@projectatomic). You can also use the [atomic-devel](https://lists.projectatomic.io/mailman/listinfo/atomic-devel) mailing list. Contributions are welcome.
+For now, you can ask us questions in the #atomic channel on IRC Freenode or via Twitter [@projectatomic](https://twitter.com/projectatomic). You can also use the [https://lists.projectatomic.io/mailman/listinfo/atomic-devel](atomic-devel) mailing list. Contributions are welcome.
```
Comment (dustymabe) [inline source/docs/fedora-atomic-workstation.md]: so, now we have undone #520, can you fix it back to `[atomic-devel](https://lists.projectatomic.io/mailman/listinfo/atomic-devel)`

Comment (dustymabe) [reply source/docs/fedora-atomic-workstation.md]: 😃

**Comment by dustymabe** [comment]
LGTM

**Pull request merged**